fourcc.py

```diff
@@ -2,57 +2,82 @@
 
 import sys
 import re
+from itertools import chain, combinations
 
-fourcc_include = sys.argv[1]
+def powerset(s):
+	return list(chain.from_iterable(combinations(s, r) for r in range(len(s) + 1)))
```

Comment:
No need to convert to a list, we can just use the iterator directly

Reply:
Fixed.